README.md

```diff
@@ -55,8 +87,8 @@ This FastAPI, React, MongoDB repo will generate a complete web application stack
   - **Standards-based**: Based on (and fully compatible with) the open standards for APIs: [OpenAPI](https://github.com/OAI/OpenAPI-Specification) and [JSON Schema](http://json-schema.org/).
   - [**Many other features**]("https://fastapi.tiangolo.com/features/"): including automatic validation, serialization, interactive documentation, etc.
 - [**Nextjs/React**](https://nextjs.org/) frontend:
-  - **Authorisation** via middleware for page access, including logged in or superuser.
-  - **Form validation** with [Vee-Validate 4](https://vee-validate.logaretm.com/v4/).
+  - **Authorization** route-based authentication, including support for detecting a user's logged in or superuser.
```

Comment:
"support for detecting a user's logged in or superuser." -> "support for detecting if a user is logged in or is a superuser."

Reply:
Removed the change here and will explicitly resolve in #14